polyfills/Intl/~locale/af-NA/config.json

```diff
@@ -1,28 +1,29 @@
 {
-	"browsers": {
-		"ie": "<11",
-		"ie_mob": "10",
-		"firefox": "<29",
-		"opera": "<15",
-		"chrome": "<24",
-		"safari": "*",
-		"ios_saf": "*",
-		"firefox_mob": "*"
-	},
-	"dependencies": [
-		"Intl"
-	],
-	"build": {
-		"minify": false
-	},
-	"license": "MIT",
-	"spec": "http://www.ecma-international.org/ecma-402/1.0/",
-	"repo": "https://github.com/andyearnshaw/Intl.js",
-	"docs": "https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl",
-	"notes": [
-		"Locales must be specified separately by prefixing the locale name with `Intl.~locale`, eg `Intl.~locale.en-GB`."
-	],
-	"test": {
-		"ci": false
-	}
-}
+    "browsers": {
+        "android": "<=4.3",
+        "ie": "<11",
+        "ie_mob": "10",
+        "firefox": "<29",
+        "opera": "<15",
+        "chrome": "<24",
+        "safari": "*",
```

Comment:
Right now the Safari TP is still using 9.x, but my guess is that the Safari version number will be 10 once this is released to the general public.